std/traits.d

```diff
@@ -1382,8 +1382,8 @@ unittest
     static assert(functionAttributes!(S.sharedF) == (FA.shared_ | FA.system));
     static assert(functionAttributes!(typeof(S.sharedF)) == (FA.shared_ | FA.system));
 
-    static assert(functionAttributes!(S.refF) == (FA.ref_ | FA.system));
-    static assert(functionAttributes!(typeof(S.refF)) == (FA.ref_ | FA.system));
+    static assert(functionAttributes!(S.refF) == (FA.ref_ | FA.system | FA.return_));
+    static assert(functionAttributes!(typeof(S.refF)) == (FA.ref_ | FA.system | FA.return_));
```

Comment:
Until the compiler fix applied, this fixed test case cannot pass tester. However until this test fix applied, the compiler fix won't pass tester. We won't merge RED PRs, so this naive fix does not work.

To avoid the mutual dependency between dmd and Phobos, you can add a temporary workaround here to accept the compiler fix first, like:

```d
  static if ([__traits(getFunctionAttributes, S.refF)] == ["ref", "@system"]))
  {
    // Bugzilla 14874: __traits(getFunctionAttributes) does not support the new `return` attribute
    static assert(functionAttributes!(S.refF) == (FA.ref_ | FA.system));
    static assert(functionAttributes!(typeof(S.refF)) == (FA.ref_ | FA.system));
  }
  else
  {
    static assert(functionAttributes!(S.refF) == (FA.ref_ | FA.system | FA.return_));
    static assert(functionAttributes!(typeof(S.refF)) == (FA.ref_ | FA.system | FA.return_));
  }
```

After the compiler actually fixed, we can remove the unneeded workaround.